### ISSUE TITLE
Open email under Decisions tab is not getting updated for a capaign

### ISSUE THREAD

**Issue by rnagabandi** [mautic/mautic#884]
Steps:

1) created lead , list , email and campaign
2) After running campaign
3) tried to check campaign status

Actual:

It shows correctly in Details graphical section, where as Open email is not updated , always it showing as 0% though it send some emails. Whereas in actions tab it has correct data.

Could you please check and help us on this?

**Comment by alanhartless** [comment]
Could you provide a screenshot of your campaign? This is sounding more like a setup issue rather than a bug. 

Thanks!

**Comment by alanhartless** [comment]
Actually; I think this is a bug that happens to be fixed with https://github.com/mautic/mautic/pull/906/files#diff-56f43beb27337325294f6460a7c3a1baR1191.  If you want to test it, I wouldn't apply that entire PR yet as it's a new feature but you could try to edit app/bundles/CampaignBundle/Model/EventModel.php line 1082 down into the else statement so that the section of code looks like something like (lines 1078-1084):

```
                                    if ($response === false) {
                                        $repo->deleteEntity($log);
                                        $logger->debug('CAMPAIGN: ID# '.$e['id'].' execution failed.');
                                    } else {
                                        $logDecision = true;
                                        $logger->debug('CAMPAIGN: ID# '.$e['id'].' successfully executed and logged.');
```

**Comment by alanhartless** [comment]
Fixed and will be part of 1.2.1.
